modules/op-setting-eventlistener-scc.adoc

```diff
@@ -0,0 +1,43 @@
+// This module is included in the following assemblies:
+// * secure/securing-webhooks-with-event-listeners.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="op-setting-eventlistener-scc_{context}"]
+= Configuring security context for event listeners
+
+You can configure a custom security context directly in your `EventListener` resource definition to meet your security requirements. A custom security context can help ensure that containers run with restricted privileges and comply with {OCP} security context constraints (SCCs).
+
+.Procedure
+
+* Create a YAML file that defines your `EventListener` CR:
+
```

Comment:
Technically needs a `+` here so that the YAML itself lines up with the bulleted step, but I'm not going to hold up a merge review for that. You can address it in the future if you want.